tests/integration/backward_compatible/hazelcast_dbapi20_test.py

```diff
@@ -0,0 +1,61 @@
+from hazelcast.config import Config
+import tests.integration.backward_compatible.dbapi20 as dbapi20
+from hazelcast import db
+from .sql_test import SqlTestBase
+
+
+class test_HazelcastDBAPI20(SqlTestBase, dbapi20.DatabaseAPI20Test):
```

Comment:
Can we use a more conventional name for the class?

Reply:
https://github.com/hazelcast/hazelcast-python-client/pull/586/commits/6f0929e54cd6bac8cc755b1bae86fc73245df4a1